locales/es.yaml

```diff
@@ -99,23 +99,23 @@ sfc:
   ModeSelector:
     gridMode: Vista en cuadrícula
     tableMode: Vista detallada
-    filterMode: Modo de filtrado
+    filterMode: Modo filtrado
     myListMode: MI LISTA
     copyFilterLink: Copiar los filtros actuales como enlace
     noFilterWarn: Seleccione primero al menos un filtro.
     exportMyList: Exportar MI LISTA a un archivo
-    myListEmptyWarn: "¡MI LISTA está vacía!\nNo hay nada que exportar."
+    myListEmptyWarn: "¡MI LISTA está vacía!\No hay nada que exportar."
```

Comment:
`\n` is the newline character. Keep it as-is.

Reply:
Sorry, that was just a misdelete